app/views/declaration/consignee_details.scala.html

```diff
@@ -31,17 +31,18 @@
 
```

Comment:
The back link should point to `exporter-details` if I see correctly.

Reply:
I have addressed this :) Thank you!